control/passivity.py

```diff
@@ -0,0 +1,66 @@
+'''
+Author: Mark Yeatman  
+Date: May 15, 2022
+'''
+
+import numpy as np
+
+try:
+    import cvxopt as cvx
+except ImportError as e:
+    cvx = None
+
+
+def is_passive(sys):
+    '''
+    Indicates if a linear time invarient system is passive
```

Comment:
`invarient` -> `invariant`

I'm just lurking around so I'm not too sure about PR etiquette. Hopefully this is okay feedback.

Reply:
I appreciate it!